pkg/apis/serving/v1alpha2/explainer.go

```diff
@@ -23,7 +23,7 @@ import (
 type Explainer interface {
 	GetResourceRequirements() *v1.ResourceRequirements
 	GetStorageUri() string
-	CreateExplainerContainer(modelName string, predictorHost string, config *InferenceServicesConfig) *v1.Container
+	CreateExplainerContainer(modelName string, parallelism int, predictorHost string, config *InferenceServicesConfig) *v1.Container
```

Comment:
I think we probably just need to start passing the whole spec in. This is growing. Feel free to punt for now.